### DISCUSSION TITLE
关于无用的外部配置文件问题

### DISCUSSION THREAD

**Discussion by Nine499** [MetaCubeX/mihomo#1079] [Q&A]
rule-providers和proxy-providers功能
如果我更改了这两个的url地址，或者直接删除一些规则引用或者是订阅引用，导致已经下载的外部资源文件有被任何配置文件使用，内核没有自动删除。请问有什么办法吗？

**Comment by Nine499** [comment]
clash meta for android 也有这个问题。

**Comment by wyf9** [comment]
貌似没有很好的解决方法，手动删除对应配置？
